### PR TITLE
Bg/add gddr7

### PR DESCRIPTION
**Pull request by bhusang**

### CODE REVIEW

File: src/protocols/memory/gddr7.stanza [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  port DQE             ; PAM3 or NRZ
+  port ERR
+
+public pcb-bundle gddr7-control-channel :
+  port RESET_n         ; NRZ
+  port ZQ_AB
+  port ZQ_CD
+
+
+doc: \<DOC>
```
Comment (callendorph) [inline src/protocols/memory/gddr7.stanza:54]: This docstring doesn't seem to be associated with anything. Is this right ?

Comment (callendorph) [reply src/protocols/memory/gddr7.stanza:54]: Perhaps it should go in the package level docstring ?

File: src/protocols/memory/gddr7.stanza [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    default => 5.0
+  doc: \<DOC>
+  Differential Routing Structure for each Diff-Pair
+  <DOC>
+  diff-route-struct:DifferentialRoutingStructure 
+  doc: \<DOC>
+  Single-Ended Routing Structure for each SE signal
+  <DOC>
+  se-route-struct:RoutingStructure
+with:
```
Comment (callendorph) [inline src/protocols/memory/gddr7.stanza:180]: Let's add `printer => true` here.

File: examples/protocols/memory/gddr7/gddr7-src.stanza [hunk 3]
```diff
@@ -0,0 +1,649 @@
...
+  num-leads = 900,
+  rows = 30,
+  columns = 30,
+  lead-diam = 0.25,
+  pitch = 0.75,
+  package-body = ic-pkg-body,
+  density-level = DensityLevelB
+)
+
+defn lookup-bga-row-name (idx:Int, table:Tuple<Char>) :
```
Comment (callendorph) [inline examples/protocols/memory/gddr7/gddr7-src.stanza:517]: There is a lot of code related to letter row lookup etc in `numbering.stanza`:

https://github.com/JITx-Inc/jsl/blob/88bf97f505ed0ebd74da9c6d418fd3d6eb102b9f/src/landpatterns/numbering.stanza#L471

Are we sure we can't reuse that ?

**Review by callendorph — APPROVED** [review]
:+1: Looks good. Sorry for all the spelling mistakes!

File: src/protocols/memory/gddr7.stanza (hunk 1 above)
Comment (bhusang) [reply src/protocols/memory/gddr7.stanza:54]: Done

**Pull request merged**